cmd/soroban-cli/src/main.rs

```diff
@@ -19,10 +19,10 @@ const HEADING_RPC: &str = "OPTIONS (RPC)";
 #[derive(Parser, Debug)]
 #[clap(
     name = "soroban",
-    version,
+    version = Box::leak(Box::new(version::short())).as_str(),
+    long_version = Box::leak(Box::new(version::long())).as_str(),
```

Comment:
Annoyingly these have to be leaked so they are `'static` lifetime for clap.

Reply:
they can't be callables? :/

Reply:
Unfortunately, clap wants a `&'help str` :(

Well.. it actually wants an `IntoResettable<Str>`  (https://docs.rs/clap/latest/clap/builder/struct.Command.html#method.version), so we could build a struct that implements that, or is convert-able to Str or something, but... I figured it wasn't worth it for now.